spec/section_spec.rb

```diff
@@ -4,37 +4,56 @@
 
 describe SitePrism::Page do
   describe '.section' do
+    class Page < SitePrism::Page; end
+    class Section < SitePrism::Section; end
+
+    class PageWithAnonymousSection < SitePrism::Page
```

Comment:
Each of these is only used in one of the contexts below.  It might make sense to move them down to the context in which they're relevant.